### DISCUSSION TITLE
記事にvimeoの動画も埋め込めるようにしてほしい

### DISCUSSION THREAD

**Discussion by kazhashimoto** [increments/qiita-discussions#278] [要望・アイデア・不具合]
記事にvimeoの動画もiframeタグで埋め込めるようにしてほしいです。
現状、記事の編集画面にvimeoの埋め込みタグをコピペしても結果はテキストのリンクしか表示されません。

CodePen経由でVimeo Player APIを呼び出す方法でCodePenを埋め込めばvimeo動画も再生できますが、
異様な方法なので直接貼り付けたいです。
参考） https://qiita.com/kazhashimoto/items/232670180415593a225c

**Comment by tanaka-asami415** [comment]
@kazhashimoto 
ご意見ありがとうございます！🙏
埋め込みについては、他にも様々なご意見をいただいておりますので
今回のご意見も合わせて検討させていただきます🙇
